packages/zapp/console/src/components/Launch/LaunchForm/utils.ts

```diff
@@ -190,3 +191,16 @@ export function isEnterInputsState(state: BaseInterpretedLaunchState): boolean {
     LaunchState.SUBMIT_SUCCEEDED,
   ].some(state.matches);
 }
+
+export function parseMappedTypeValue(value: InputValue): { key: string; value: string } {
+  try {
+    const mapObj = JSON.parse(value.toString());
+    const mapKey = Object.keys(mapObj)?.[0] || '';
```

Comment:
It's better to use `??` notation instead, as 
`const mapKey = Object.keys(mapObj)?.[0] ?? '';`

There are situations where || leads in such cases leads to very well hidden errors. I don't have an example from the top of my head. But I ca fins one if needed.

You are using it in few places here and in file above to default to an empty string value. Please replace.

Reply:
fixed